src/Parsers/New/AST/AlterTableQuery.h

```diff
@@ -88,19 +94,19 @@ class AlterTableClause : public INode
     private:
         enum ChildIndex : UInt8
         {
-            // ADD COLUMN or INDEX
+            // ADD COLUMN, INDEX or PROJECTION
             ELEMENT = 0,  // TableElementExpr
```

Comment:
If you use this value for some other clause types, then please mark in comments all possible AST classes, like `Identifier`, etc.